service.toml

```diff
@@ -36,10 +36,32 @@ optional = ["list_mode"]
 required = ["expire"]
 
 [namespace.storage.op.read]
-optional = ["offset", "io_callback", "size"]
+optional = ["offset", "io_callback", "size", "sse_customer_algorithm", "sse_customer_key", "sse_customer_key_md5"]
 
 [namespace.storage.op.write]
-optional = ["content_md5", "content_type", "io_callback", "storage_class"]
+optional = ["content_md5", "content_type", "io_callback", "storage_class", "sse_customer_algorithm", "sse_customer_key", "sse_customer_key_md5"]
+
+[namespace.storage.op.copy]
+optional = ["sse_customer_algorithm", "sse_customer_key", "sse_customer_key_md5", "sse_copy_source_customer_algorithm", "sse_copy_source_customer_key", "sse_copy_source_customer_key_md5"]
+
+[pairs.sse_customer_algorithm]
+type = "string"
+
+[pairs.sse_customer_key]
```

Comment:
Let's add some `description` here, how to construct a valid key? What value we need here?